apollo-router/tests/integration/telemetry/otlp.rs

```diff
@@ -208,7 +347,7 @@ async fn test_untraced_request_sample_datadog_agent_unsampled() -> Result<(), Bo
 #[tokio::test(flavor = "multi_thread")]
 async fn test_priority_sampling_propagated() -> Result<(), BoxError> {
     if !graph_os_enabled() {
-        return Ok(());
+        panic!("Error: test skipped because GraphOS is not enabled");
```

Comment:
This will mean users that don't have a graphOS key will not be able to have passing tests